aptos-move/framework/src/natives/aptos_lib.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use move_binary_format::errors::PartialVMResult;
+use move_core_types::{
+    gas_schedule::GasCost,
+    language_storage::{StructTag, TypeTag},
+};
+use move_vm_runtime::native_functions::NativeContext;
+use move_vm_types::{
+    loaded_data::runtime_types::Type, natives::function::NativeResult, values::Value,
+};
+use smallvec::{smallvec, SmallVec};
+use std::{collections::VecDeque, fmt::Write};
+
+/// Return Token types ModuleAddress, ModuleName and StructName
```

Comment:
nit: It's not just for Token, this could be used for any Type?

Reply:
mahahahahahhahahaha I did not copy this code...